### DISCUSSION TITLE
Change admin ui route

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#3983] [Q&A]
Is it possible to change the admin dashboard UI route to another route?

For example:
http://127.0.0.1:8090/_/ --->  http://127.0.0.1:8090/test/

**Comment by ganigeorgiev** [comment]
The admin ui route is not configurable at the moment and cannot be changed (at least not without route rewrites or other hacky workaround).

Proper support for this will be added sometime in the future but there are not ETAs and remains a low priority.

**Comment by darktempla** [comment]
You could look at using Nginx proxy in front of PB.
